tests/bears/DebugBearsTest.py

```diff
@@ -60,3 +75,28 @@ def test_run_return_generator_with_debugger(self):
         self.assertEqual(lines[3], '-> yield 1')
         self.assertEqual(lines[5], '-> yield 2')
         self.assertEqual(lines[7], '-> yield 3')
+
+    def test_do_settings(self):
+        self.section.append(Setting('x', '85'))
+        self.section.append(Setting('y', 'kbc3'))
+        self.section.append(Setting('z', 86))
+        my_bear = TestOneBear(self.section, self.queue)
+        args = ('a', ('b', 'c'))
+        kwargs = {'x': 2, 'y': 'abc'}
+        result, output = execute_debugger(["settings x=3; y= 'abc2'; z=abc;"
+                                           'some_key=2', 'q', 'settings', 'c',
+                                           'q'], my_bear.run, *args, **kwargs)
+        # lines = output.splitlines()
+        print(output)
```

Comment:
Test are failing because of 3 different result for same ouput variable (travis one is correct)
--> Circle Ci
```
-> yield 2
(Pdb) w = 'kbc'
y = 'abc2'
z = 86
x = 3
```
--> Appveyor
```
-> yield 2
(Pdb) w = 'kbc'
z = 86
x = 3
y = 'abc2'
```
--> Travis
```
-> yield 2
(Pdb) x = 3
y = 'abc2'
z = 86
w = 'kbc'
```

Reply:
To prevent the CI failure this code can be changed to 
```
lines = output.splitlines()
self.assertEqual(lines[2], "(Pdb) name 'abc' is not defined")
self.assertEqual(lines[3], "'some_key' key is not in scope")
settings_list = ['x = 3', "y = 'abc2'", 'z = 86', "w = 'kbc'"]
for i in settings_list:
    self.assertIn(i, output)
```